server/cli/start.go

```diff
@@ -169,6 +160,7 @@ func runStart(cmd *commander.Command, args []string) {
 	select {
 	case <-stopper.ShouldStop():
 		stopper.SetStopped()
+		close(gracefulShutdownCh)
```

Comment:
We can't close this yet. You need to wait until all the Stopper.closers have been called.
Two options (at least):
- have stopper.Stop() be callable multiple times (you would need the mutex to be held on the closer runners and probably a noop check at the beginning)
- have a new channel to signal that Stop() has finished (Stopper.stopper is not enough, we need the closers too)